ci-operator/step-registry/optional-operators/cvp-common/aws/optional-operators-cvp-common-aws-workflow.yaml

```diff
@@ -10,6 +10,7 @@ workflow:
     - ref: optional-operators-subscribe
     test:
     - ref: optional-operators-cvp-common-test
+    - ref: optional-operators-cvp-common-scorecard
```

Comment:
We need to run this test before the `optional-operators-cvp-common-test` because that one runs additional checks after all of the testing is done.

Reply:
Changing the order.